### PR TITLE
Bug 1866548: Update Cluster Settings unit test to reflect new design changes

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1866548

### CODE REVIEW

**Comment by rhamilto** [comment]
This is my first pass at writing a significant chunk of new unit tests.  Definitely needs a thorough look.

File: frontend/public/module/k8s/types.ts [hunk 1]
```diff
@@ -719,6 +721,7 @@ export type MachineConfigPoolKind = {
 } & K8sResourceKind;
 
 export type ClusterUpdate = {
+  force: boolean;
```
Comment (spadgett) [inline frontend/public/module/k8s/types.ts:724]:
```suggestion
  force?: boolean;
```

**Pull request merged**